### ISSUE TITLE
Environment test fails, not able to decrypt secrets

### ISSUE THREAD

**Issue by prachiN48** [codecentric/gopass-ui#46]
MacOS version : 10.15.3
Gopass version : 1.8.6
Gopass UI version : 0.6.0
I am able to see the store and the list of it's contents but unable to see individual secrets. when I clcik on any entry in store the UI hangs on loading screen

<img width="554" alt="Screenshot 2020-03-25 at 18 19 57" src="https://user-images.githubusercontent.com/62475407/77565941-40067880-6ec5-11ea-9bd8-eab2bbd7c298.png">

**Comment by ruettenm** [comment]
Are you using **pinentry-mac** as a GPG passphrase dialog tool?

https://formulae.brew.sh/formula/pinentry-mac

**Comment by prachiN48** [comment]
Yes, I am and now after a machine restart the environment test has worked but it’s still unable to retrieve the secret values..
The screen looks something like this.

I’ve highlighted the error in the Developer tools in the bottom right corner.

**Comment by lunny** [comment]
I also have the problem with gopass 1.8.6 and gopass-ui 0.7.0

**Comment by Saggitarie** [comment]
Still having this error. 
OS: Mac 11.3.1

**Comment by deligatedgeek** [comment]
I had this for some time, but after much playing I realised that I had not configured the PINENTRY env
`PINENTRY=$(which pinentry-mac)`

It works for me once set
